website/content/en/preview/getting-started-with-terraform/_index.md

```diff
@@ -241,29 +265,44 @@ Use helm to deploy Karpenter to the cluster. We are going to use the
 `helm_release` Terraform resource to do the deploy and pass in the cluster
 details and IAM role Karpenter needs to assume.
 
+Add the following to your `main.tf` to provision Karpenter via a Helm chart.
+
 ```hcl
+provider "helm" {
+  kubernetes {
+    host                   = module.eks.cluster_endpoint
+    cluster_ca_certificate = base64decode(module.eks.cluster_certificate_authority_data)
+
+    exec {
+      api_version = "client.authentication.k8s.io/v1alpha1"
+      command     = "aws"
+      args        = ["eks", "get-token", "--cluster-name", local.cluster_name]
+    }
+  }
+}
+
 resource "helm_release" "karpenter" {
-  depends_on       = [module.eks.kubeconfig]
   namespace        = "karpenter"
   create_namespace = true
 
   name       = "karpenter"
   repository = "https://charts.karpenter.sh"
   chart      = "karpenter"
-  version    = "{{< param "latest_release_version" >}}"
+  # Be sure to pull latest version of chart
+  version = "0.6.1"
 
   set {
     name  = "serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn"
     value = module.iam_assumable_role_karpenter.iam_role_arn
   }
 
   set {
-    name  = "clusterName"
-    value = var.cluster_name
+    name  = "controller.clusterName"
```

Comment:
the latest version of the helm chart removed the `controller` prefix from these since its actually passed to both the controller and webhook container and these two containers are now colocated into 1 pod. https://github.com/aws/karpenter/tree/main/charts/karpenter

Reply:
ah, ok that makes sense now. updated to point to latest release (0.6.3)